P5/Source/Specs/gender.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="gender">
+  <gloss xml:lang="en" versionDate="2022-05-03">gender</gloss>
+  <desc versionDate="2022-05-17" xml:lang="en">specifies the gender identity of a person, persona, or character.</desc>
+  <desc versionDate="2022-05-03" xml:lang="es">especifica la identidad de género de una persona.</desc>
+  <desc versionDate="2022-05-03" xml:lang="it">specifica l'identità di genere di una persona.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.persStateLike"/>
+    <memberOf key="att.editLike"/>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.typed"/>
+  </classes>
+  <content>
+    <macroRef key="macro.phraseSeq"/>
+  </content>
+  <attList>
+    <attDef ident="type" usage="opt" mode="change">
```

Comment:
A bit confused — what is the point of lines 26–30? Since this is a member of att.typed it already has a `@type` attribute that is teidata.enumerated without a value list.

Reply:
Indeed this is an error due to using the specs of `<sex>` as a template. In `<sex>` there was the sample values “explicit” and “implicit”, and indeed, if we are not going to suggest any values for `@type`, this should be deleted.

Reply:
Fixed with https://github.com/TEIC/TEI/pull/2299/commits/1bd0f6a5ffa6712e20c0e545881b5755eff0073b